### PR TITLE
Fix session filters not updating

### LINKED ISSUE #289: [PE-D] "list <argument>" command should be handled.
Is `list <argument>` command allowed in the app? Because there is nothing mentioned about it in the user guide. It does not handle unknown command error, and instead, the output shows `Listed all sessions`.

-------------
Labels: `severity.Low` `type.FunctionalityBug`
original: mario7lorenzo/ped#2

### LINKED ISSUE #342: Listing sessions does not clear filters in column titles

### PR DESCRIPTION
**Pull request by PotatoCombat**

### CODE REVIEW

File: src/main/java/tatracker/commons/core/Messages.java [hunk 1]
```diff
@@ -11,7 +11,7 @@
     public static final String MESSAGE_INVALID_COMMAND_FORMAT = "Invalid command format!\n%1$s";
     public static final String MESSAGE_INVALID_COMMAND = "Invalid command format!\n\n";
 
-    public static final String MESSAGE_UNKNOWN_COMMAND = "Unknown command";
+    public static final String MESSAGE_UNKNOWN_COMMAND = "Unknown command\n\n";
```
Comment (fatin99) [inline src/main/java/tatracker/commons/core/Messages.java:14]: why are there two line breaks? :O Was that intentional?

File: src/main/java/tatracker/logic/commands/session/FilterSessionCommand.java [hunk 2]
```diff
@@ -31,12 +31,14 @@
             MODULE, DATE, SESSION_TYPE
     );
 
-    public static final String MESSAGE_FILTERED_SESSIONS_SUCCESS = "Filtered Session List: %s";
```
Comment (fatin99) [inline src/main/java/tatracker/logic/commands/session/FilterSessionCommand.java:34]: Why did you change the success message tho? i thought it was nice that the user gets to see what filters have just been applied

File: src/main/java/tatracker/ui/claimstab/ClaimsCard.java [hunk 3]
```diff
@@ -43,7 +43,7 @@
     private Label description;
 
     private final DateTimeFormatter timeFormat = DateTimeFormatter.ofPattern("hh:mma");
-    private final DateTimeFormatter dateFormat = DateTimeFormatter.ofPattern("MMM d");
+    private final DateTimeFormatter dateFormat = DateTimeFormatter.ofPattern("dd MMM yyyy");
```
Comment (fatin99) [inline src/main/java/tatracker/ui/claimstab/ClaimsCard.java:46]: Thanks!

File: src/main/java/tatracker/ui/sessiontab/SessionCard.java [hunk 4]
```diff
@@ -45,7 +45,7 @@
     private Label recur;
 
     private final DateTimeFormatter timeFormat = DateTimeFormatter.ofPattern("hh:mma");
-    private final DateTimeFormatter dateFormat = DateTimeFormatter.ofPattern("MMM d");
+    private final DateTimeFormatter dateFormat = DateTimeFormatter.ofPattern("dd MMM yyyy");
```
Comment (fatin99) [inline src/main/java/tatracker/ui/sessiontab/SessionCard.java:48]: Thanks!

File: src/main/java/tatracker/logic/commands/session/FilterSessionCommand.java (hunk 2 above)
Comment (PotatoCombat) [reply src/main/java/tatracker/logic/commands/session/FilterSessionCommand.java:34]: Yup it does! Now instead it shows only the ones that have been specified.
Before it did this: Date | Module | Type

Now its
Date:
Module:
Type:

File: src/main/java/tatracker/commons/core/Messages.java (hunk 1 above)
Comment (PotatoCombat) [reply src/main/java/tatracker/commons/core/Messages.java:14]: Yep! It's for spacing out the messages.
I'm assuming that you always have two line breaks when concatenating another message.

File: src/main/java/tatracker/logic/commands/session/FilterSessionCommand.java (hunk 2 above)
Comment (PotatoCombat) [reply src/main/java/tatracker/logic/commands/session/FilterSessionCommand.java:34]: Can test then will see!

**Review by aakanksha-rai — APPROVED** [review]
LGTM!

**Pull request merged**